README.md

```diff
@@ -89,6 +89,17 @@ $ make build
 $ ./svc-test
 ```
 
+# Run with Sonobuoy
+```
+$ brew instal sonobuoy
```

Comment:
what if we are not on macosx? can we at least flag this is the macosx command to install it.

Reply:
refined the instructions.